conformance/tests/httproute-request-mirror.yaml

```diff
@@ -7,38 +7,7 @@ spec:
   parentRefs:
   - name: same-namespace
   rules:
-  - matches:
-    - path:
-        type: PathPrefix
-        value: /mirror
-    filters:
-    - type: RequestMirror
-      requestMirror:
-        backendRef:
-          name: infra-backend-v2
-          namespace: gateway-conformance-infra
-          port: 8080
-    backendRefs:
-    - name: infra-backend-v1
-      port: 8080
-      namespace: gateway-conformance-infra
-  - matches:
-    - path:
-        type: PathPrefix
-        value: /mirror-and-modify-headers
-    filters:
```

Comment:
why rm this case thats testing the filter combo ?

Reply:
For BackendRef modifying headers and mirroring are two separate features